### PR TITLE
Patchset 202104

### PR DESCRIPTION
**Pull request by rusxakep**
- Remove lot legacy bash code
- LOGROTATE_FORCE CaSe fix
- ZABBIX_REMOTECOMMANDS_ALLOW mispelling fix

### CODE REVIEW

**Comment by tiredofit** [comment]
This is great stuff. I'm going to take a bit to go through the `01-permissions` stuff as its pretty fragile, but just from a quick glance it looks like you've really shored it up!

I hope to merge this within 24 hours.

File: Dockerfile [hunk 1]
```diff
@@ -54,7 +54,7 @@ RUN set -x && \
         armel) s6Arch='arm' ;; \
         armhf) s6Arch='armhf' ;; \
 		arm64) s6Arch='aarch64' ;; \
-		ppc64le) s6Arch='ppc64le' ;; \
+		ppc64el) s6Arch='ppc64le' ;; \
```
Comment (tiredofit) [inline Dockerfile:57]: Have a peek at this: https://lists.debian.org/debian-powerpc/2014/08/msg00042.html

Comment (tiredofit) [reply Dockerfile:57]: This is what QEMU in Github Actions is capable of doing:

````
| 2021/04/25 21:59:50 installing: mips64el qemu-mips64el already registered
| {
|   "supported": [
|     "linux/amd64",
|     "linux/arm64",
|     "linux/riscv64",
|     "linux/ppc64le",
|     "linux/s390x",
|     "linux/386",
|     "linux/mips64le",
|     "linux/mips64",
|     "linux/arm/v7",
|     "linux/arm/v6"
|   ],
|   "emulators": [
|     "CLR",
|     "qemu-aarch64",
|     "qemu-arm",
|     "qemu-mips64",
|     "qemu-mips64el",
|     "qemu-ppc64le",
|     "qemu-riscv64",
|     "qemu-s390x"
|   ]
| }
````

Comment (rusxakep) [reply Dockerfile:57]: > This is what QEMU in Github Actions is capable of doing:
> 
> ```
> | 2021/04/25 21:59:50 installing: mips64el qemu-mips64el already registered
> | {
> |   "supported": [
> |     "linux/amd64",
> |     "linux/arm64",
> |     "linux/riscv64",
> |     "linux/ppc64le",
> |     "linux/s390x",
> |     "linux/386",
> |     "linux/mips64le",
> |     "linux/mips64",
> |     "linux/arm/v7",
> |     "linux/arm/v6"
> |   ],
> |   "emulators": [
> |     "CLR",
> |     "qemu-aarch64",
> |     "qemu-arm",
> |     "qemu-mips64",
> |     "qemu-mips64el",
> |     "qemu-ppc64le",
> |     "qemu-riscv64",
> |     "qemu-s390x"
> |   ]
> | }
> ```

Problem not with Github QEMU emulator, but with debian 'dpkg --print-architecture'. It report ppc64**el** on ppc64le QEMU emulator.

**Pull request closed without merging**